src/ibex_bluesky_core/plans/__init__.py

```diff
@@ -12,15 +12,15 @@
 from ophyd_async.plan_stubs import ensure_connected
 
 from ibex_bluesky_core.callbacks import ISISCallbacks
-from ibex_bluesky_core.devices.block import BlockMot
+from ibex_bluesky_core.devices.block import BlockMot, BlockRw, BlockWriteConfig
 from ibex_bluesky_core.devices.simpledae import monitor_normalising_dae
 from ibex_bluesky_core.fitting import FitMethod
 from ibex_bluesky_core.utils import NamedReadableAndMovable, centred_pixel, get_pv_prefix
 
 if TYPE_CHECKING:
     from ibex_bluesky_core.devices.simpledae import SimpleDae
 
-__all__ = ["adaptive_scan", "motor_adaptive_scan", "motor_scan", "polling_plan", "scan"]
+__all__ = ["BlockMot", "adaptive_scan", "motor_adaptive_scan", "motor_scan", "polling_plan", "scan"]
```

Comment:
```suggestion
__all__ = ["adaptive_scan", "motor_adaptive_scan", "motor_scan", "polling_plan", "scan"]
```

I don't think `BlockMot` should be exported here (people should be importing that from `devices.block`

Reply:
thanks pycharm